src/pkg/bundle/bundle.go

```diff
@@ -81,7 +82,37 @@ func Create(b *Bundler, signature []byte) error {
 					}
 					// ensure media type is Zarf blob for layers in the bundle's root manifest
 					layerDesc.MediaType = oci.ZarfLayerMediaTypeBlob
+
+					// add package name annotations
+					annotations := make(map[string]string)
+					layerDesc.Annotations = annotations
+					layerDesc.Annotations[config.UDSPackageNameAnnotation] = pkg.Name
+
+					// If zarf package name has been obtained from zarf config, set the zarf package name annotation
+					if zarfPackageName != "" {
```

Comment:
Won't `zarfPackageName` always be `""`? I see on line 55 where we define it to be an empty string, but I don't see where that would change before it hits line 92